spec/view_helper/applinks_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe MetaTags::ViewHelper, 'displaying Applinks meta tags' do
+  subject { ActionView::Base.new }
+
+  it 'should display meta tags specified with :al' do
+    subject.set_meta_tags(al: {
+      foo:       'bar',
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.